### DISCUSSION TITLE
Lesson 12 import file error

### DISCUSSION THREAD

**Discussion by PavanTamrakar** [Cyfrin/foundry-full-course-cu#194] [Q&A] [answered]
![Screenshot (180)](https://github.com/Cyfrin/foundry-full-course-f23/assets/72463719/3f05a0b5-7f01-4dd4-ad4a-7a448b49353e)

I did everything right but still got this error. I checked for spelling mistakes and everything. Can someone please help?! @PatrickAlphaC

**Comment by ChefAharoni** [comment, marked as answer]
VS Code is a little bit buggy with Foundry.

Please provide a SC or copy&paste your foundry.toml file.

A few ways to solve it:

1. Comment and un-comment the import lines. Helps "refresh" the import.

2. Restart your VS Code completely, including all opened windows. Worked for me a couple of times.

3. Try creating a 'remappings.txt' file, and add the same line without brackets and "".

**Reply by PavanTamrakar** [reply]
This is what I get while compiling it
The file path is correct

![Screenshot (181)](https://github.com/Cyfrin/foundry-full-course-f23/assets/72463719/238838d2-ec58-4039-9d11-baa459adf486)

**Comment by PavanTamrakar** [comment]
Creating a 'remappings.txt' file with "" worked. Thank you.
